nav2_bt_navigator/behavior_trees/navigate_w_replanning_and_recovery.xml

```diff
@@ -20,12 +20,12 @@
       </PipelineSequence>
       <ReactiveFallback name="RecoveryFallback">
         <GoalUpdated/>
-        <SequenceStar name="RecoveryActions">
+        <RoundRobin name="RecoveryActions">
           <ClearEntireCostmap name="ClearLocalCostmap-Subtree" service_name="local_costmap/clear_entirely_local_costmap"/>
           <ClearEntireCostmap name="ClearGlobalCostmap-Subtree" service_name="global_costmap/clear_entirely_global_costmap"/>
           <Spin spin_dist="1.57"/>
           <Wait wait_duration="5"/>
```

Comment:
We should add back up recovery here if this is the default tree we plan to use

Reply:
Sure, do we want backup _instead_ of spin? Or both? If both, are you OK with backup and _then_ spin?

Reply:
I think back up should be the last option. This is where we have all the recoveries available and each recovery is tried one by one until one succeeds. Back up seems the most drastic option so having it at the end of this list makes more sense.

Reply:
I can also change to the `ForceSuccess` decorator node if the switch to `RoundRobin` is not desired

Reply:
Ah, I made a mistake here (read my last comment on the PR). Revert the changes to the BT please. Go back to using `SequenceStar` and remove `BackUp` from the sequence.